content/desktop/synchronized-file-sharing.md

```diff
@@ -83,19 +65,19 @@ You can use a `.syncignore` file at the root of each file share, to exclude loca
  
 Some example of things you might want to add to your `.syncignore` file are:
 - Large dependency directories, for example `node_modules` and `composer` directories (unless you rely on accessing them via a bind mount)
-- `.git directories` (again, unless you need them)
+- `.git` directories (again, unless you need them)
 
 In general, use your `.syncignore` file to exclude items that aren't critical to your workflow, especially those that would be slow to sync or use significant storage.
 
 ## Frequently asked questions
 
-### What is the difference between Synchronized file sharing and the Mutagen extension?
+### What is the difference between Synchronized File Shares and the Mutagen extension?
 
-Since Docker [acquired Mutagen](https://www.docker.com/blog/mutagen-acquisition/), Synchronized file sharing is essentially the direct integration of the Mutagen extension into Docker Desktop. However, Synchronized file sharing provides an improved user experience and the ability to use a [`.syncignore` file](#use-syncignore). You also no longer have to set the default ownership on files as this is now handled automatically.
+Since Docker [acquired Mutagen](https://www.docker.com/blog/mutagen-acquisition/), Synchronized File Shares is essentially the direct integration of the Mutagen extension into Docker Desktop. However, Synchronized File Shares provides an improved user experience and the ability to use a [`.syncignore` file](#use-syncignore). You also no longer have to set the default ownership on files as this is now handled automatically.
 
 ## Known issues
 
-- Upon launching Docker Desktop, it can take between 5-10 seconds for Synchronized file sharing to fully initialize. During this window, file share instances display as **Connecting** and any new containers created during this window won't replace bind mounts with Synchronized file sharing.
+- Upon launching Docker Desktop, it can take between 5-10 seconds for Synchronized File Shares to fully initialize. During this window, file share instances display as **Connecting** and any new containers created during this window won't replace bind mounts with Synchronized File Shares.
```

Comment:
Given some recent improvements, I think we can remove this issue.  There's still a delay, but it should now be almost imperceptible.